maintaining.rst

```diff
@@ -0,0 +1,276 @@
+
+Generally, releasing in the collections consists of:
+
+  1. Planning and announcement.
+  2. Generating a changelog.
+  3. Creating a release git tag and pushing it.
+  4. Automatic publishing the release tarball on `Ansible Galaxy <https://galaxy.ansible.com/>`_ by Zuul.
+  5. Final announcement.
+
+For more information about the process, refer to the `Releasing guidelines <releasing.rst>`_.
+
+.. _Expanding community:
+
+Expanding community
+===================
+
+Increasing a number of active contributors and maintainers
+----------------------------------------------------------
+
+Maintainers are interested in increasing a number of active long-term contributors for a collection they maintain.
+
+Contributors are reviewers, issue or pull request authors, testers, maintainer, and all other people who help develop the project.
+
+Every regular contributor once was a newcomer. Make the first experience as positive as possible to make the new people coming back.
+
+Good development documentation makes contributors life much easier. Get feedback from new contributors if there were things they struggled with when working on their proposals and improve the documentation correspondingly.
+
+Create the ``CONTRIBUTING`` file in your repository. In there, add a link to the `Quick-start guide <create_pr_quick_start_guide.rst>` as well as to other guidelines describing things specific to your collection.
+
+Make contributors feel welcome. Greet and thank contributors impersonally in ``README`` and individually in their proposals.
+Thank all participants after merging or closing a proposal.
+
+Be responsive. Respond as quickly as possible. Even if you cannot review a proposal right now, greet and thank the author.
+
+Add and keep updated the ``CONTRIBUTORS`` file listing all the contributors including issue reporters and refer to it from your ``README``.
+
+Do not fix trivial non-critical bugs yourself. Instead, mentor a person who would like to contribute.
+Mark issues with labels like ``easy_fix``, ``help_wanted``, and ``documentation``.
+They will let newcomers know where they can find easy wins.
+
+When reviewing an issue, if applicable, ask the author whether they want to fix the issue themselves providing the link to the `Quick-start guide <create_pr_quick_start_guide.rst>`.
+
+Adopt a zero-tolerance policy towards behavior violating `Ansible Code of Conduct <https://docs.ansible.com/ansible/latest/community/code_of_conduct.html>`_. Add information to ``README`` how people can complain referring to the `policy-violations Code of Conduct section<https://docs.ansible.com/ansible/latest/community/code_of_conduct.html#policy-violations>`_.
+
+Announce that the project needs new contributors and maintainers through available communication channels.
+
+Promote active contributors satisfying :ref:`requirements<Requirements for maintainers>` to maintainers. Revise contributors activity regularly.
+
+Create the ``MAINTAINERS`` file and keep it updated.
```

Comment:
Do we have this file in any repository yet? I have only seen `commit-rights.md` file in c.g.

Reply:
I've recently added it to c.postgresq, c.proxysql, c.mysql and links from their READMEs to it. See https://github.com/ansible-collections/community.mysql. Though it doesn't mean I insist to have it here applied all across ansible-collections organization:)